Administrator Guide/GlusterFS Introduction.md

```diff
@@ -1,63 +1,33 @@
-Introducing Gluster File System
-===============================
-
-GlusterFS is an open source, distributed file system capable of scaling to
-several petabytes and handling thousands of clients. It is a file system with
-a modular, stackable design, and a unique no-metadata server architecture.
-This no-metadata server architecture ensures better performance,
-linear scalability, and reliability. GlusterFS can be
-flexibly combined with commodity physical, virtual, and cloud resources
-to deliver highly available and performant enterprise storage at a
-fraction of the cost of traditional solutions.
-
-GlusterFS clusters together storage building blocks over Infiniband RDMA
-and/or TCP/IP interconnect, aggregating disk and memory resources and
-managing data in a single global namespace.
-
-GlusterFS aggregates various storage servers over network interconnects
-into one large parallel network file system. Based on a stackable user space
-design, it delivers exceptional performance for diverse workloads and is a key
-building block of GlusterFS.
-The POSIX compatible GlusterFS servers, use any ondisk file system which supports
-extended attributes (eg: ext4, XFS, etc) to format to store data on disks, can be
-accessed using industry-standard access protocols including Network File System (NFS)
-and Server Message Block (SMB).
+What is Gluster ?
+=================
 
-![640px-glusterfs_architecture](https://cloud.githubusercontent.com/assets/10970993/7412187/dbb94286-ef5d-11e4-93ca-d4959a94db85.png)
+Gluster is a scalable, distributed file system that aggregates disk and memory resources into a single global namespace.
```

Comment:
The term "memory resources" can be misleading. It's better to change it to `disk storage` or `disk storage capacity` or even just `disks`

Reply:
Will do.